SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```

Comment:
Great idea on adding this!

Due to the way the `changeset` versioning package works, I can only maintain the latest major version of electron-builder/updater

Please update this version table to:
```
electron-builder

| Version | Supported          |
| ------- | ------------------ |
| 24.x    | :white_check_mark: |
| < 24.0  | :x:                |


electron-updater

| Version | Supported          |
| ------- | ------------------ |
| 6.x     | :white_check_mark: |
| < 6.0   | :x:                |
```